diff --git a/node_modules/@arbitrum/orbit-sdk/dist/chains.js b/node_modules/@arbitrum/orbit-sdk/dist/chains.js
index 9af8c16..8cc495e 100644
--- a/node_modules/@arbitrum/orbit-sdk/dist/chains.js
+++ b/node_modules/@arbitrum/orbit-sdk/dist/chains.js
@@ -65,6 +65,7 @@ exports.chains = [
     // testnet
     chains_1.sepolia,
     chains_1.arbitrumSepolia,
+    chains_1.baseSepolia,
     // local nitro-testnode
     nitroTestnodeL1,
     nitroTestnodeL2,
diff --git a/node_modules/@arbitrum/orbit-sdk/dist/generated.js b/node_modules/@arbitrum/orbit-sdk/dist/generated.js
index 72ca4da..187ebb9 100644
--- a/node_modules/@arbitrum/orbit-sdk/dist/generated.js
+++ b/node_modules/@arbitrum/orbit-sdk/dist/generated.js
@@ -654,6 +654,7 @@ exports.tokenBridgeCreatorAddress = {
     1337: '0x54B4D4e578E10178a6cA602bdb6df0F213296Af4',
     42161: '0x2f5624dc8800dfA0A82AC03509Ef8bb8E7Ac000e',
     42170: '0x8B9D9490a68B1F16ac8A21DdAE5Fd7aB9d708c14',
+    84532: "0xFC71d21a4FE10Cc0d34745ba9c713836f82f8DE3",
     333333: '0x0000000000000000000000000000000000000000',
     412346: '0x38F35Af53bF913c439eaB06A367e09D6eb253492',
     421614: '0x56C486D3786fA26cc61473C499A36Eb9CC1FbD8E',
